.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: regression
+    attributes:
+      label: Regression
+      description: Is the reported bug a regression? If so, what is the last version of ROS where it still worked fine?

```

We can probably make it more general field to fill in.

```suggestion
    id: description
    attributes:
      label: Description
      description: Explain what the problem is. Besides, is the reported bug a regression? If so, what is the last version of ROS where it still worked fine?
```